views/security/login.php

```diff
@@ -33,37 +35,46 @@
             </div>
             <div class="panel-body">
                 <?php $form = ActiveForm::begin([
-                    'id'                     => 'login-form',
-                    'enableAjaxValidation'   => true,
+                    'id' => 'login-form',
+                    'enableAjaxValidation' => true,
                     'enableClientValidation' => false,
-                    'validateOnBlur'         => false,
-                    'validateOnType'         => false,
-                    'validateOnChange'       => false,
+                    'validateOnBlur' => false,
+                    'validateOnType' => false,
+                    'validateOnChange' => false,
                 ]) ?>
 
-                <?= $form->field(
-                    $model,
-                    'login',
-                    ['inputOptions' => ['autofocus' => 'autofocus', 'class' => 'form-control', 'tabindex' => '1']]
-                ) ?>
+                <?php
```

Comment:
Could you please change this to short tags (<?= instead of <?php echo and <?php if(): ?> instead of <?php if {} ?>.